components/omega/OmegaBuild.cmake

```diff
@@ -71,6 +71,10 @@ macro(read_cime_config)
     set(NEWCASE_COMMAND "${NEWCASE_COMMAND} --compiler ${OMEGA_CIME_COMPILER}")
   endif()
 
+  if(NOT "${OMEGA_CIME_PROJECT}" STREQUAL "")
+    set(NEWCASE_COMMAND "${NEWCASE_COMMAND} --project ${OMEGA_CIME_PROJECT}")
+  endif()
+
```

Comment:
@grnydawn, an alternative approach would just be to pass a dummy `--project omega` or something to `create_newcase` in every case, since we won't actually use the project for anything.